awscli/customizations/s3/subcommands.py

```diff
@@ -218,6 +218,19 @@
 }
 
 
+SSE_KMS_ENCRYPTION_CONTEXT = {
+    'name': 'sse-kms-encryption-context',
+    'help_text': (
```

Comment:
I copied the help text for the `--ssekms-encryption-context` arg for the `s3api put-object` command